packages/stacks-docs/product/foundation/typography.html

```diff
@@ -208,11 +210,10 @@
 {% endhighlight %}
         <div class="stacks-preview--example d-flex fd-column gy16 my0">
             {% for term in typography.fontsize %}
-            <div class="d-flex">
-                <div class="flex--item3 fc-light fs-caption fw-bold">{{ term.label }}</div>
-                <div class="flex--item fs-{{ term.term }} lh-xs">
-                    <span class="d-block sm:d-none">{{ term.define }}</span>
-                    <span class="d-none sm:d-block">{{ term.responsive }}</span>
+            <div class="d-flex my16">
+                <div class="flex--item3 fl-shrink0 fc-light fs-caption fw-bold">{{ term.label }}</div>
+                <div class="flex--item fs-{{ term.term }}">
+                    <span class="d-block sm:d-none hmx2 overflow-hidden">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</span>
```

Comment:
I wonder if we should consider using one of our `v-truncate` classes here. I honestly thought there was a rendering issue when I initially viewed this section 😂 

```suggestion
                    <span class="v-truncate3">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</span>
```

Reply:
Makes a lot of sense